container/test-app/wc/clientAPI.html

```diff
@@ -43,6 +45,8 @@ <h3>
       import MFEventID from '../bundle.js';
 
       [...document.querySelectorAll('luigi-container')].forEach(luigiContainer => {
+        luigiContainer.init();
```

Comment:
This line might be excessive. The luigi container in this file would need `.init() `function to be initialised only if it had the **defer-init** attribute set to true. If this is needed regardless, then we might have a bug.